docs/usage/remote-plugins.rst

```diff
@@ -45,3 +46,46 @@ but it can make asynchronous requests, i.e. passing ``async=True``.
 
 You need to run ``:UpdateRemotePlugins`` in Neovim for changes in the specifications to have effect.
 For details see ``:help remote-plugin`` in Neovim.
+
+If you are working on your plugin locally, it may be desirable to run the
+plugin from a git repository directly without the need of installing or
+symlinking it.
```

Comment:
This kind of thing does not need to be spelled out in extreme detail, we have to assume some basic level of familiarity with software development. So we really need to keep the documents concise, to avoid reader fatigue.

Reply:
I'll be honest, this is something I prefer myself: having documentation as clear as possible. I don't like when I have to read between the lines and figure out details myself the hard way (such as, how do I make nvim pick up my plugin from `$PWD`?).

I completely understand if you don't prefer this style and I have no problem adapting. That's why we do these pull requests and reviews, right?